tests/security/test_dependency_security.py

```diff
@@ -7,6 +7,8 @@
 import numpy as np
 import pandas as pd
 
+from packaging.version import Version
```

Comment:
I would suggest to remove this new package. It adds a new dependency, and it doesn't add too much value.

In the past, we had a lot of problems when adding extra dependencies, I think we should try to minimize external dependencies as much as possible

Reply:
I've checked that **packaging** has no extra dependencies.  See [packaging/pyproject.toml](https://github.com/pypa/packaging/blob/main/pyproject.toml)

Reply:
Because packaging is only used for testing, do you have any suggestion on where to packaging.  @miguelgfierro   I saw you added [docs/requirements-doc.txt](https://github.com/recommenders-team/recommenders/blob/staging/docs/requirements-doc.txt) for doc dependencies.

Reply:
> Because packaging is only used for testing, do you have any suggestion on where to packaging

In the past, adding extra dependencies gave us a lot of extra work. For example, the issue we had with snapbook #2012 and for example, a new one we need to fix with pandera #2049. 

Even though this new library doesn't have additional dependencies, it is still one extra dependency. In addition, it is just simpler to do the comparison like this `torch.__version__ >= "1.13.1"` than `Version(torch.__version__) >= Version("1.13.1")`

I think we should only add a new dependency if there is no other alternative and it is really necessary. That would save us a lot of work.

Does it make sense @SimonYansenZhao?

Reply:
Their dependencies are minimal, see https://github.com/pypa/packaging/blob/4d8534061364e3cbfee582192ab81a095ec2db51/pyproject.toml, so I don't think we should worry so much about this package causing problems.

Reply:
ok, let's hope this doesn't break in a few years